reproducibility_index

```diff
@@ -0,0 +1,10 @@
+61fdf25b4e2b4171a579dd9e4716b24348fb3f9f90ad936f9c91046618c93ed5  ./target/wasm32-unknown-unknown/release/abitest_0_frontend.wasm
```

Comment:
Should it be in a separate `artifact` directory?

Reply:
- we don't have an `artifact` folder (yet)
- if we had one, its contents would not be checked in, but this file should still be checked in

so I would leave this here for now, but happy to restructure things more generally at some point